components/steps/CheckedIn.tsx

```diff
@@ -127,17 +127,13 @@ const CheckedIn: React.FunctionComponent<Props> = props => {
         </MarginedColumn>
         <MarginedColumn flexBasis={65}>
           <CheckInTitle>Tasks To-Do</CheckInTitle>
-          <CheckInInstructions>
```

Comment:
lol so apparently CheckInInstructions was a `styled.p` and so we were getting warnings about headers being a child component of a `<p>` tag